torch/onnx/symbolic_helper.py

```diff
@@ -950,29 +963,58 @@ def dequantize_helper(g, qtensor, qdtype=None):
     value = g.op("Cast", tensor, to_i=qdtype)
     scale = g.op("Cast", scale, to_i=torch.onnx.TensorProtoDataType.FLOAT)
     zero_point = g.op("Cast", zero_point, to_i=qdtype)
-    return g.op("DequantizeLinear", value, scale, zero_point), scale, zero_point
 
-def quantize_helper(g, tensor, scale, zero_point):
+    if _export_onnx_opset_version < 13:
```

Comment:
Suggestion for making the control flow a bit clearer here. 

```python
axis_kwargs = {}
if axis_i:
  if _export_onnx_opset_version < 13:
     _onnx_opset_unsupported_detailed(...)
   axis_kwargs = {"axis_i": axis_i}
```

Reply:
Related to the other comment, can simplify condition to one line now that we don't need axis_kwargs.